tests/infra/repos/postgresql/guardian-account.spec.ts

```diff
@@ -16,28 +16,38 @@ describe('GuardianAccountRepository', () => {
     const guardianData = {
       firstName: 'valid_first_name',
       lastName: 'valid_last_name',
-      email: 'valid_email',
+      email: 'any_email@mail.com',
```

Comment:
You could create a method to create the guardianData and the guardian that is expected in the tests

Reply:
This is already being implemented by @SamuelS00, mocks and stubs files.